usaspending_api/disaster/v2/views/disaster_base.py

```diff
@@ -49,6 +52,30 @@ def filter_by_latest_closed_periods() -> Q:
     return q
 
 
+def filter_by_defc_closed_periods() -> Q:
+    """
+        These filters should only be used when looking at submission data
+        that includes DEF Codes, which only started appearing in submission
+        for FY2020 P07 (Apr 1, 2020) and after
+    """
+    q = Q()
+    for sub in final_submissions_for_all_fy():
+        if (
+            sub.fiscal_year == REPORTING_PERIOD_MIN_YEAR
+            and sub.fiscal_period >= REPORTING_PERIOD_MIN_MONTH
+        ) or sub.fiscal_year > REPORTING_PERIOD_MIN_YEAR:
+            q |= (
+                Q(submission__reporting_fiscal_year=sub.fiscal_year)
+                & Q(submission__quarter_format_flag=sub.is_quarter)
+                & Q(submission__reporting_fiscal_period__lte=sub.fiscal_period)
+            )
+    if not q:
+        # Edgecase not expected in production. If there are no DABS between
+        # FY2020 P07 (Apr 1, 2020) and now() then ensure nothing is returned
+        q = Q(pk__isnull=True)
```

Comment:
@kwhickey Here's the fix to handle when there are no closed periods between 2020/04/01 - present. I think it breaks some tests (need to confirm it was this)